daprdocs/content/en/reference/components-reference/supported-pubsub/_index.md

```diff
@@ -1,9 +1,11 @@
 ---
 type: docs
 title: "Supported pub/sub brokers"
-linkTitle: "Supported pub/sub brokers"
-weight: 30000
+linkTitle: "Pub/sub brokers"
```

Comment:
I think we may want to also remove the "supported" from the title field (can keep the folder name, not to make this harder). Not needed in the new structure and when seeing all of them together in the top level index I think it is redundant.

Reply:
Hmm I'm still partial to including "Supported" because this page shows a list of all the pub/sub brokers that Dapr supports, instead of a page *on* pub.sub brokers as a concept

Reply:
Updated to "Pub/sub brokers component specs", with similar titles for other building blocks